src/assets/scripts/client/canvas/CanvasController.js

```diff
@@ -841,25 +841,16 @@ export default class CanvasController {
     _drawSingleRadarTarget(cc, radarTargetModel) {
         const { aircraftModel } = radarTargetModel;
         const match = prop.input.callsign.length > 0 && aircraftModel.matchCallsign(prop.input.callsign);
+        let fillStyle = this.theme.RADAR_TARGET.HISTORY_DOT_OUTSIDE_RANGE;
 
         if (!aircraftModel.isVisible()) {
             return;
         }
 
-        // Trailling
-        let trailling_length = this.theme.RADAR_TARGET.HISTORY_LENGTH;
-        const dpr = window.devicePixelRatio || 1;
-
-        if (dpr > 1) {
-            trailling_length *= round(dpr);
-        }
-
         cc.save();
 
-        let fillStyle = this.theme.RADAR_TARGET.HISTORY_DOT_INSIDE_RANGE;
-
         if (!aircraftModel.inside_ctr) {
-            fillStyle = this.theme.RADAR_TARGET.HISTORY_DOT_OUTSIDE_RANGE;
+            fillStyle = this.theme.RADAR_TARGET.HISTORY_DOT_INSIDE_RANGE;
```

Comment:
definitely still want `HISTORY_DOT_OUTSIDE_RANGE` here

Reply:
🤔 hmm, not sure why I changed that one

Reply:
oh, I see I [was supposed to] flipped the logic